packages/graphql/server.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import '@babel/polyfill';
+
+import { ApolloServer } from 'apollo-server';
+
+import createContext from './src/services/GraphQLContext';
+import schema from './src/Schema';
+
+const server = new ApolloServer({ schema, context: createContext() });
+
+server.listen().then(({ url, context }) => {
+  console.log(`🚀 Server ready at ${url}`); // eslint-disable-line no-console
+});
```

Comment:
Why are you starting the server? Is there something wrong with the previous (literally) server-less approach? :-/

Reply:
IMHO the ability to use tools like GraphiQL

Reply:
GraphiQL doesn't need the network to get GraphQL schema. 🤔

Reply:
Well, out of curiosity, I would like to know how to use GraphiQL without a server running, because normally, I feed it an url. Not sure how to use it without an url to feed it.

Reply:
I tried it run with Playground React Component but I didn't find information in the documentation how to do it without the server.
So I decided to make graphql server. 

I from start expected that there will be the point when we need to split the app.

Reply:
@tbergq You have to use `schema` property and not `fetcher` property.

> `schema`: a GraphQLSchema instance or `null` if one is not to be used. If `undefined` is provided, GraphiQL will send an introspection query using the fetcher to produce a schema.

https://github.com/graphql/graphiql#usage

> I from start expected that there will be the point when we need to split the app.

Yes, this is how I designed this _server-less_ GraphQL as well. Just a question - got the answer, let's move on... :)